sdk/core/core-amqp/test/config.spec.ts

```diff
@@ -91,7 +91,68 @@ describe("ConnectionConfig", function () {
         "Endpoint=sb://localhost:6765;SharedAccessKeyName=<< REDACTED >>;SharedAccessKey=<< REDACTED >>;UseDevelopmentEmulator=true",
       );
       assert.equal(config.endpoint, "sb://localhost:6765/");
-      assert.equal(config.host, "localhost:6765");
+      assert.equal(config.host, "localhost");
+      assert.equal(config.port, 6765);
+      assert.isTrue(config.useDevelopmentEmulator);
+    });
+
+    it("Parses the connection string for 127.0.0.1", async function () {
+      const config = ConnectionConfig.create(
+        "Endpoint=sb://127.0.0.1:6765;SharedAccessKeyName=<< REDACTED >>;SharedAccessKey=<< REDACTED >>;UseDevelopmentEmulator=true",
+      );
+      assert.equal(config.endpoint, "sb://127.0.0.1:6765/");
+      assert.equal(config.host, "127.0.0.1");
+      assert.equal(config.port, 6765);
+      assert.isTrue(config.useDevelopmentEmulator);
+    });
+
+    it("Parses the connection string for 127.0.0.1", async function () {
+      const config = ConnectionConfig.create(
+        "Endpoint=sb://127.0.0.1:6765;SharedAccessKeyName=<< REDACTED >>;SharedAccessKey=<< REDACTED >>;UseDevelopmentEmulator=true",
+      );
+      assert.equal(config.endpoint, "sb://127.0.0.1:6765/");
+      assert.equal(config.host, "127.0.0.1");
+      assert.equal(config.port, 6765);
+      assert.isTrue(config.useDevelopmentEmulator);
+    });
```

Comment:
Are some of these tests duplicate of each other?

Reply:
Good catch! I meant to remove the port for one of them.

Reply:
Fixed